src/index.js

```diff
@@ -1,6 +1,8 @@
 import "./index.scss";
 
 import { AppComponent } from "./components/app/app";
+import { PieChartComponent } from "./components/pie-chart/pie-chart";
```

Comment:
you should import it in app for now, but in ToolbarComponent content in future. we don't change index.js at all. I think you do not have to change it now, because you will need to change it soon again